nx-dev/ui-terms/src/lib/terms-and-definitions.tsx

```diff
@@ -0,0 +1,510 @@
+      <h2>1. Definitions</h2>
+      <dl>
+        <dd>
+          (a) &quot;<strong>Aggregate Data</strong>&quot; means any data that is
+          derived or aggregated in deidentified form from (i) Your Data; or (ii)
+          you and your Authorized Users’ use of the Software, including, without
+          limitation, any usage data, metrics, metadata or trends with respect
+          to the Software (including the workspaces therein) and its component
+          features and functionalities, in each case in anonymized form.
+        </dd>
+        <dd>
+          (b) &quot;<strong>Authorized User</strong>&quot; means any employee or
+          contributor that you authorize to Use the Software for purposes of
+          operating and maintaining the Software.
+        </dd>
+        <dd>
+          (c) &quot;<strong>Documentation</strong>&quot; means the operator and
+          user manuals, training materials, specifications, minimum system
+          configuration requirements, and other similar materials in hard copy
+          or electronic form if and as provided by Nx to Customer (including any
+          revised versions thereof) relating to the Software.
+        </dd>
+        <dd>
+          (d) &quot;<strong>Licensed Volume</strong>&quot; means the limits,
+          volume or other measurement or conditions of permitted Use for the
+          Software as set forth in the Order Information, including any limits
+          on the number of Authorized Users permitted to Use the Software based
+          on your subscription tier.
+        </dd>
+        <dd>
+          (e) &quot;<strong>Nx IP</strong>&quot; means the Software, the
+          underlying software provided in conjunction with the Software,
+          algorithms, interfaces, technology, databases, tools, know-how,
+          processes and methods used to provide or deliver the Software or any
+          related services, Documentation and Aggregate Data, all improvements,
+          modifications or enhancements to, or derivative works of, the
+          foregoing (regardless of inventorship or authorship), and all
+          intellectual property rights in and to any of the foregoing.
+        </dd>
+        <dd>
+          (f) &quot;<strong>Order Information</strong>&quot; means certain terms
+          associated with your subscription to Use the Software, as communicated
+          to you at the time you signed up for a subscription electronically
+          through our Site via our Pricing Page.
+        </dd>
+        <dd>
+          (g) &quot;<strong>Person</strong>&quot; means any individual,
+          corporation, partnership, trust, limited liability company,
+          association, governmental authority or other entity.
+        </dd>
+        <dd>
+          (h) &quot;<strong>Pricing Page</strong>&quot; means our Pricing Page
+          available at https://nx.dev/pricing, which may be updated from time to
```

Comment:
@juristr this was changed from `nx.app/pricing` is this fine from a legal standpoint?